### PR TITLE
Update list-of-acme-servers.md

### PR DESCRIPTION
**Pull request by aelliotr**
Added SSL.com to list of publicly trusted CAs.

### CODE REVIEW

File: list-of-acme-servers.md [hunk 1]
```diff
@@ -27,4 +27,7 @@ Please note that different CAs have varying legal terms, pricing, and some diffe
   * Production:
     * OV RSA: [https://acme.sectigo.com/v2/InCommonRSAOV](https://acme.sectigo.com/v2/InCommonRSAOV)
     * OV ECC: [https://acme.sectigo.com/v2/InCommonECCO](https://acme.sectigo.com/v2/InCommonECCOV)
-
+* [**SSL.com**](https://www.ssl.com/blogs/sslcom-supports-acme-protocol-ssl-tls-certificate-automation/)
```
Comment (mholt) [inline list-of-acme-servers.md]:
```suggestion
* [**SSL.com**](https://www.ssl.com/guide/ssl-tls-certificate-issuance-and-revocation-with-acme/)
```

This page has more useful information.

**Review by mholt — CHANGES_REQUESTED** [review]
Thank you for the PR!

Do you work with SSL.com / are you a customer? I noticed that the ACME directories say `"externalAccountRequired":false` but that does not seem to be accurate; the documentation seems to indicate otherwise, and my own attempt at using the endpoint fails without EAB.

File: list-of-acme-servers.md (hunk 1 above)
Comment (aelliotr) [reply list-of-acme-servers.md]: Hey there! I work at SSL.com but am the content manager. Checking in with the ACME team on your question.

**Comment by aelliotr** [comment]
> Thank you for the PR!
> 
> Do you work with SSL.com / are you a customer? I noticed that the ACME directories say `"externalAccountRequired":false` but that does not seem to be accurate; the documentation seems to indicate otherwise, and my own attempt at using the endpoint fails without EAB.

According to our ACME developers, the intended behavior is to handle the absence of EAB, and the implementation of a solution is imminent.

**Comment by mholt** [comment]
Great; I'll merge this in as documentation and leave it up to your team to address that issue. Thanks!

**Review by mholt — APPROVED** [review]

**Comment by aelliotr** [comment]
Thanks!

**Pull request merged**